File: packages/insomnia-httpsnippet/test/fixtures/output/java/okhttp/headers.java
OkHttpClient client = new OkHttpClient();

Request request = new Request.Builder()
  .url("http://mockbin.com/har")
  .get()
  .addHeader("accept", "application/json")
  .addHeader("x-foo", "Bar")
  .build();

Response response = client.newCall(request).execute();

File: packages/insomnia-httpsnippet/test/fixtures/output/node/request/headers.js
var request = require('request');

var options = {
  method: 'GET',
  url: 'http://mockbin.com/har',
  headers: { 'x-foo': 'Bar', accept: 'application/json' }
};

request(options, function(error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});
